torch/distributed/optim/apply_optimizer_in_backward.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Dict, Iterable, Type
+
+import torch
+
+__all__ = []  # type: ignore[var-annotated]
```

Comment:
nit: I wonder if we just annotated this as `__all__: List[str] = []`, we can remove the `# type: ignore[var-annotated]`.